src/reformat_libyuv.c

```diff
@@ -746,6 +929,23 @@ avifResult avifImageYUVToRGBLibYUV(const avifImage * image, avifRGBImage * rgb)
                                                             image->width,
                                                             image->height,
                                                             filter);
+    } else if (lcf.yuvaToRgbMatrixFilterHighBitDepth != NULL) {
+        libyuvResult = lcf.yuvaToRgbMatrixFilterHighBitDepth((const uint16_t *)image->yuvPlanes[AVIF_CHAN_Y],
+                                                             image->yuvRowBytes[AVIF_CHAN_Y] / 2,
+                                                             (const uint16_t *)image->yuvPlanes[uPlaneIndex],
+                                                             image->yuvRowBytes[uPlaneIndex] / 2,
+                                                             (const uint16_t *)image->yuvPlanes[vPlaneIndex],
+                                                             image->yuvRowBytes[vPlaneIndex] / 2,
+                                                             (const uint16_t *)image->alphaPlane,
+                                                             image->alphaRowBytes / 2,
+                                                             rgb->pixels,
+                                                             rgb->rowBytes,
+                                                             matrix,
+                                                             image->width,
+                                                             image->height,
+                                                             /*attenuate=*/0,
```

Comment:
Vignesh: I just realized an important issue. Is it correct to always pass /*attenuate=*/0 to these functions? The `attenuate` parameter is equivalent to our alphaPremultiplied boolean.

If it is correct, it would be good to add a comment to reassure the readers.

Reply:
Yes, we are basically using libyuv to do the same thing that's being done in lines 1260-1282 in `reformat.c`. That operation requires passing 0 to `attenuate`. If there is a premultiply, that is handled separately in lines 1370-1380 in `reformat.c`